Updates to War Screens, Macrobuilder, Ages and more

Xyarvius opened the pull request:
Hi @Glidar-dev @2kProjectEu4,

i had some time to make some changes again. Mostly small changes where i center elements, strings etc. but also some bigger changes.

**Here are the bigger ones:**
![p2kuie](https://github.com/user-attachments/assets/030658ba-1544-4c4a-a038-16a1cc3dfb6b)


**Colored Mapmodes**
The changes for the colored mapmodes submod are in the branch "colored_minimap_icons":
https://github.com/2kProjectEu4/Proper-2K-UI-Enhanced/tree/colored_minimap_icons
It is is just an upscale of this mod:
https://steamcommunity.com/sharedfiles/filedetails/?id=3390117335&searchtext=mapmode
The original mod is not needed as it also only contains the .dds files.

**Fix for "P2KE Patch - Plati's Decision Pictures"**
You can download the changes which fix how Platis decisions are displayed in tooltips and in the history here:
https://drive.google.com/file/d/1bTYxONuA1uU5hBiEx9bCoj4j9g-Laf4p/view?usp=sharing

**Change for "P2KE Submod - Tech and Ideas Pictures"**
You can download the changes make the images in the idea select window bigger, which imo fits better with how they are displayed in the idea overview window, here:
https://drive.google.com/file/d/1eTTH02jJTIVAAaTWnWpCEPD2rPgz_EeR/view?usp=sharing

All the changes are, as always, documented here:
https://docs.google.com/document/d/1g3vGjARjzKosMFv1WKSP0l-643D-pq3yFXXCLizH3Y4/edit?tab=t.0#heading=h.tib9zsaongs4

2kProjectEu4 commented:
Hello @Xyarvius, it looks really good. I will take a look and publish the changes in the following days.

Xyarvius commented:
@2kProjectEu4 Awesome.

I also just finished an adaption of Platis Ages mod for a P2kUIE-submod. I made a couple of bigger layout changes, primarily to stay closer to how the Ages view is designed in the P2KUIE base mod. I also added pictures, similar to ones used in TGE and added some missing tooltips.

The files are located in the branch "platis_ages_submodule": 
https://github.com/2kProjectEu4/Proper-2K-UI-Enhanced/tree/platis_ages_submodule
Or you can download them here:
https://drive.google.com/file/d/1VSc4gJA7_aCmY4ssV29-ufbS8H27uS1E/view?usp=sharing

![{910111CC-6692-4B62-A903-FE5A3D24600D}](https://github.com/user-attachments/assets/9a7c08a3-5a40-469a-bed8-f2c35d2f9665)

![{C470981D-8055-4BC2-9CBB-F6884DB2DC61}](https://github.com/user-attachments/assets/8f52690a-76c9-4770-a74b-4059bd1f5ae5)

![{4EAD91E4-8860-481F-BDF1-6FC507D0F14C}](https://github.com/user-attachments/assets/47c962f7-f709-417a-b1a6-a0911bb28956)

![{A1C4438E-1737-4BEB-AFA2-1E2CE7A695A1}](https://github.com/user-attachments/assets/7a4cca33-2455-4f04-a122-e422f8a6b11a)

2kProjectEu4 commented:
Cool, I’ll take a look at the two new submods over the weekend, I think.
Regarding the pull request, from what I tested, everything looks fine. I only found one small issue in the macrobuilder: the list of provinces when you build is overlapping a bit with the new separator.
![Captură de ecran 2025-04-09 233253](https://github.com/user-attachments/assets/10d09e21-568c-4cb4-ab3f-0df287a891bc)

Xyarvius commented:
Thanks, i totally missed the province building list. I updated it:
![{9F56FE0B-D814-4BA0-8D58-8581D2F4AA33}](https://github.com/user-attachments/assets/1043569b-e7d8-43ce-a013-e7f0f401b302)

Xyarvius commented:
@2kProjectEu4  Nice, good that you noticed this. 
I increased the width of the decisions window and banners slightly, moved the two buttons a bit to the right and moved the checkbox + added a background for it:
![{EF9AAB97-0CC3-4EBD-B3C6-72AB3530B676}](https://github.com/user-attachments/assets/6823aa5f-eab2-4ac4-971e-94271c988390)


I don't know if this is enough for every decision string but since "Yangban Administrators" seems to be one of the longest estate strings, i am positive that it might be. What do you think?

Here are the changes: https://drive.google.com/file/d/1gk60Q4vM-z1GL-pfmdfofIvy5V50CHk8/view?usp=sharing

2kProjectEu4 commented:
@Xyarvius Its perfect. Thank you.
